nncf/openvino/quantization/accuracy_aware.py

```diff
@@ -0,0 +1,77 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+from functools import partial
+
+import numpy as np
+from addict import Dict
+
+from openvino.tools import pot
+
+
+class ExperimentalAccuracyAware(pot.AccuracyAwareCommon):
```

Comment:
`Experimental...` is not the right term. I would call it like `NMSEBasedAccuracyAware`

Reply:
Done